src/styles/blocks/_section-cards.scss

```diff
@@ -0,0 +1,7 @@
+.section-cards {
+  display: flex;
+  flex-wrap: wrap;
+  width: get-width(200px, 4, 48px);
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
you can make the `main` a block and the section an element of this block.